src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -262,6 +267,16 @@ export class RoslynLanguageServer {
         await this._languageClient.restart();
     }
 
+    public workspaceDisplayName(): string {
+        if (this._solutionFile !== undefined) {
```

Comment:
with devkit, you may have a solution open, but this will not be defined.  If we want to have consistent behavior in both, it would probably be better to get the item to display from the server.  As the server will know the solution name in either case.

Reply:
We hide this status item when DevKit is enabled.